commit 38e3b0e11a4f0407c870eaa9f6244d945692b490
Author: Alberto Mardegan <alberto.mardegan@canonical.com>
Date:   Mon Aug 19 15:23:59 2013 +0300

    lib/plugins: do not build a static library
    
    While there certainly were some reasons to have this library built
    statically, we have lost memory of those. It seems that building this as
    a shared library works, and plugins can be successfully linked against
    it (tested with the OAuth plugin).

diff --git a/lib/plugins/signon-plugins.pro b/lib/plugins/signon-plugins.pro
index 5a96a4f..bca4975 100644
--- a/lib/plugins/signon-plugins.pro
+++ b/lib/plugins/signon-plugins.pro
@@ -5,8 +5,6 @@ include( ../../common-project-config.pri )
 include($${TOP_SRC_DIR}/common-installs-config.pri)
 include($${TOP_SRC_DIR}/common-vars.pri)
 
-CONFIG += static
-
 HEADERS = \
     SignOn/authpluginif.h \
     SignOn/uisessiondata.h \
diff --git a/tests/run-with-signond.sh b/tests/run-with-signond.sh
index 7f0705e..3af5eaf 100755
--- a/tests/run-with-signond.sh
+++ b/tests/run-with-signond.sh
@@ -12,7 +12,7 @@ export SSO_DAEMON_TIMEOUT=5
 export SSO_IDENTITY_TIMEOUT=5
 export SSO_AUTHSESSION_TIMEOUT=5
 export PATH="${BUILDDIR}/src/remotepluginprocess:$PATH"
-export LD_LIBRARY_PATH="${BUILDDIR}/lib/plugins/signon-plugins-common":"${BUILDDIR}/lib/signond/SignOn":"$LD_LIBRARY_PATH"
+export LD_LIBRARY_PATH="${BUILDDIR}/lib/plugins":"${BUILDDIR}/lib/plugins/signon-plugins-common":"${BUILDDIR}/lib/signond/SignOn":"$LD_LIBRARY_PATH"
 export XDG_RUNTIME_DIR="${XDG_RUNTIME_DIR:-/tmp/runtime-$(whoami)}/signon-tests"
 mkdir -p "$XDG_RUNTIME_DIR"
 
